bundles/framework/publisher2/tools/LayerSelectionTool.js

```diff
@@ -88,14 +88,14 @@ function() {
         if(enabled === true) {
             me.__plugin.startPlugin(me.__sandbox);
             me.__started = true;
-            setTimeout(function(){
+            setTimeout(function() {
                 me._checkLayerSelections();
             }, 300);
         } else if(me.__started === true) {
+            me.__plugin.teardownUI();
```

Comment:
Shouldn't it be the plugins responsibility to remove it's UI when it's stopped instead of the publisher functionality?